tests/test_room_resource/test_create_room_resource.py

```diff
@@ -4,6 +4,10 @@
     resource_mutation_0_value_room_id_query, error_0_value_room_id,
     resource_mutation_empty_name_string_query, error_empty_name_string
 )
```

Comment:
@dkam26 I think some of these imports are not required and used. So they should be removed and then flake8 will pass.

Reply:
@PatrickCmd ,Thanks for the feedback.I have made the necessary changes